tests/unit/acquisition/test_function.py

```diff
@@ -312,7 +314,7 @@ def test_min_value_entropy_search_builder_builds_min_value_samples_rff(mocked_mv
     query_points = builder._search_space.sample(num_samples=builder._grid_size)
     query_points = tf.concat([dataset.query_points, query_points], 0)
     fmean, _ = model.predict(query_points)
-    assert max(min_value_samples) < min(fmean)
+    assert max(min_value_samples) < min(fmean) + 1e-4
```

Comment:
Why is this necessary (here and further down)?

Reply:
Under non-zero observations noise (with RFF sampling), we could get sample draws that were all larger than min(fmean), just by chance. As i changed the default number of min-value samples, this seemed to happen when running this test (although it didnt before).